vault/mount.go

```diff
@@ -712,13 +712,18 @@ func (c *Core) mountInternal(ctx context.Context, entry *MountEntry, updateStora
 
 // builtinTypeFromMountEntry attempts to find a builtin PluginType associated
 // with the specified MountEntry. Returns consts.PluginTypeUnknown if not found.
-func (c *Core) builtinTypeFromMountEntry(ctx context.Context, entry *MountEntry) consts.PluginType {
+func (c *Core) builtinTypeFromMountEntry(ctx context.Context, entry *MountEntry, version string) consts.PluginType {
```

Comment:
Is there an advantage to taking the version as an argument here? It looks like in both places, `entry.RunningVersion` is passed in, and since `entry` is already being passed in, you could accomplish the same change here just by e.g.

```go
version := entry.RunningVersion
```

inside this function.

Reply:
Good eye, thanks! I want to say there were more callers before I finalized the approach and just forgot to refactor, but maybe it was just an oversight. 😅